CONTRIBUTING.md

```diff
@@ -153,13 +153,27 @@ This will run a local web server with "live reload" conveniently enabled. ( **NO
   git commit -s -m "<commit subject>"
   ```
 
-  **11.** Push the committed changes in your feature branch to your remote repo.
+  **11.** While you are working on your branch, other developers may update the `master` branch with their branch. This action means your branch is now out of date of the `master` branch and missing content. So in order to fetch the new changes, follow along:
+
+  ```
+  git checkout master
+  git fetch origin master
```

Comment:
After this one should also do
```
git merge upstream/master
 git push origin
```

Reply:
@Nikhil-Ladha idk why I'm getting this error : `merge: upstream/master - not something we can merge` when I do `git merge upstream/master`

Reply:
Though, this change needs to be done.

Reply:
The output of `git remote -v` and `git status`?

Reply:
okay!

Reply:
Done!

Reply:
Are you still getting the above error?
If yes, can you share the output of above 2 commands?

Reply:
Aight, one sec.

Reply:
@Nikhil-Ladha I simply committed with my original method, so I'm getting this:
```
$ git remote -v
origin  https://github.com/Mr-DG-Wick/layer5.git (fetch)
origin  https://github.com/Mr-DG-Wick/layer5.git (push)
```

```
$ git status
On branch Contributing-doc-change
nothing to commit, working tree clean
```

Reply:
Ohh, no upstream remote is set.
Cool, then what can be done is an optional line can be added saying:
```
If you have upstream remote added, then perform the below steps:
.... (you can add those 2 steps)....
```

Reply:
Aight so shall I also add a command to set the remote reference? @Nikhil-Ladha 
that way people won't have to check for any conditions and simply follow the instructions.

Reply:
Yeah, you can add that 👍🏼